app/uk/gov/hmrc/individualincomedesstub/domain/ErrorResponse.scala

```diff
@@ -29,5 +29,10 @@ sealed abstract class ErrorResponse(
   def toHttpResponse = Results.Status(httpStatusCode)(Json.toJson(this))
 }
 
+case class ErrorInvalidRequestPayload(errorMessage: String) extends ErrorResponse(BAD_REQUEST, "INVALID_REQUEST", errorMessage)
```

Comment:
We got rid of this `ErrorInvalidRequestPayload` in the [OGD-257 commit](https://github.com/hmrc/individuals-api/commit/32856e1c9ce5ddb07f9fea8b0d3be97ba55cdff3).